root_orchestrator/system-manager-python/tests/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.3"
```

Comment:
why a compose file inside tests?

Reply:
`services_test` uses it for integration testing - since it will spin up the containers as test containers along with a mongodb test container. Also the pipeline uses it. with module `testcontainers` they provide a way to spin up containers through docker-compose file which is nice.